backend/ibutsu_server/controllers/dashboard_controller.py

```diff
@@ -51,9 +54,10 @@ def get_dashboard_list(project_id=None, user_id=None, page=1, page_size=25):
     """
     query = Dashboard.query
     if project_id:
+        print(project_id)
         query = query.filter(Dashboard.project_id == project_id)
     if user_id:
-        query = query.filter(Dashboard.user_id == user_id)
+        query = query.filter(Dashboard.user_id == UUID(user_id))
```

Comment:
Just curious, but why is this necessary here but not on L58? 
```
        query = query.filter(Dashboard.project_id == project_id)
```

Reply:
Actually, I'm not sure. I vaguely remember adding that line when trying to debug some issues, but I think it's probably not actually necessary. I'll remove it and see if I have any problems.